device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
+        "serial": "FIXME",
+        "components": [],
+        "modules": [],
+        "fans": [
+            {
+                "name": "Fan 1",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 2",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 3",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 4",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 5",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 6",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 7",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 8",
+                "direction": "intake",
+                "tolerance": 20
+            }
+        ],
+        "fan_drawers": [
+            {
+                "name": "Fan drawer 1",
+                "num_fans": 2
+            },
+            {
+                "name": "Fan drawer 2",
+                "num_fans": 2
+            },
+            {
+                "name": "Fan drawer 3",
+                "num_fans": 2
+            },
+            {
+                "name": "Fan drawer 4",
+                "num_fans": 2
+            }
+        ],
+        "psus": [
+            {
+                "name": "PSU 1",
+                "num_fans": 1,
+                "powergood_status": "true",
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2
+            },
+            {
+                "name": "PSU 2",
+                "num_fans": 1,
+                "powergood_status": "true",
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2
+            }
+        ],
+        "thermal_mgr": [
+		{
+		"FIXME":"need input from Mellanox for required static data"
+		}
+	],
+        "watchdog": [
+		{
+		"FIXME":"need input from Mellanox for required static data"
```

Comment:
what can we check for watchdog w/o arm/disarm(which covered by the watchdog test already.)?

Reply:
Agreed. This should be removed.

Reply:
fixed